### PR TITLE
CI Tweaks

### PR DESCRIPTION
**Pull request by omus**

### CODE REVIEW

**Comment by omus** [comment]
Interesting. It appears that LibCURL_jll doesn't support Linux 32-bit?

**Comment by omus** [comment]
Appveyor is expected to fail currently due to the missing Appveyor config. I'll disable the webhook for this repo if we proceed with this.

**Comment by omus** [comment]
It appears that Travis CI doesn't actually support Windows 32-bit which means I can't drop testing on Appveyor. I'll do some refactoring here as some of these changes are still useful

**Comment by omus** [comment]
Originally I planned on dropping Appveyor in favor of using Travis CI Windows. Unfortunately LibCURL doesn't work on Linux 32-bit and Travis CI does not support 32-bit Windows currently.

File: .travis.yml [hunk 1]
```diff
@@ -4,17 +4,8 @@ os:
...
-# script:
-#   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#   - julia -e 'using Pkg; Pkg.clone(pwd()); Pkg.build("LibCURL"); Pkg.test("LibCURL"; coverage=true)'
-after_success:
-  - |
-    julia -e '
-      using Pkg
-      Pkg.add("Coverage")
-      using Coverage
-      Codecov.submit(process_folder())'
```
Comment (ararslan) [inline .travis.yml:20]: Did you intend to remove coverage submission entirely?

Comment (omus) [reply .travis.yml:20]: I'll restore that. I thought that the default Travis script now did this automatically.

Comment (ararslan) [reply .travis.yml]: Nope, only if you do `codecov: true`

**Pull request merged**